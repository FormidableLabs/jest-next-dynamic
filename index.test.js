import React from "react";
import TestRenderer from "react-test-renderer";
import preloadAll from "./index";
import NextDynamicNested from "./test/NextDynamicNested";

beforeAll(async () => {
  await preloadAll();
});

it("renders nested next/dynamic components", () => {
  expect(TestRenderer.create(<NextDynamicNested />).toJSON()).toMatchSnapshot();
});
