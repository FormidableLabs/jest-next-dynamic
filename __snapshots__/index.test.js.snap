// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders nested next/dynamic components 1`] = `
<div>
  <p>
    Nested dynamic loaded component:
  </p>
  <div>
    <p>
      Dynamic loaded component:
    </p>
    <h1>
      Leaf component says: 
      Loaded with next/dynamic!
    </h1>
  </div>
</div>
`;
